app/models/vacancy.rb

```diff
@@ -78,6 +78,22 @@ class Vacancy < ApplicationRecord
 
   enum status: %i[published draft trashed]
   enum working_pattern: %i[full_time part_time]
+  enum listed_elsewhere: %i[
+    listed_paid
+    listed_free
+    listed_mix
+    not_listed
+    listed_dont_know
+  ]
+  enum hired_status: %i[
+    hired_tvs
+    hired_other_free
+    hired_paid
+    not_filled_ongoing
+    not_filled_readvertised
+    not_filled_not_looking
+    hired_dont_know
+  ]
```

Comment:
Rather than adding these to the `Vacancy`, I think these should be collected in a new linked feedback table, in a similar way to how we do `VacancyPublishFeedback`s.